tests/integration/test.replication.js

```diff
@@ -3489,6 +3714,10 @@ adapters.forEach(function (adapters) {
     });
 
     it('#3270 triggers "change" events with .docs property', function(done) {
+      if (testUtils.isSyncGateway() && adapters[0] === 'http') {
+        // SG doesn't have a replicator
+        return done();
+      }
```

Comment:
The rest of these look good; seems to be testing very CouchDB-specific auth/validation/ddoc stuff.